src/internal.c

```diff
@@ -33340,134 +33340,93 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
 #endif /* !WOLFSSL_NO_TLS12 */
 
 #ifdef HAVE_SESSION_TICKET
+    /* Make a work from the front of random hash */
+    static WC_INLINE word32 MakeWordFromHash(const byte* hashID)
```

Comment:
Now that this is in ssl.c and internal.c you might consider moving it to misc.c wrapped with `!WOLFCRYPT_ONLY` `HAVE_SESSION_TICKET`.

Reply:
Refactored `MakeWordFromHash`, `compute_InternalTicket_hash`, and `HashSession` into `misc.c`.